src/main/docs/guide/aop/validation.adoc

```diff
@@ -2,12 +2,6 @@ Validation advice is one of the most common advice types you are likely to want
 
 Validation advice is built on https://beanvalidation.org/2.0/spec/[Bean Validation JSR 380], a specification of the Java API for bean validation which ensures that the properties of a bean meet specific criteria, using `javax.validation` annotations such as `@NotNull`, `@Min`, and `@Max`.
 
-Micronaut provides native support for the `javax.validation` annotations with the `micronaut-validation` dependency:
+https://micronaut-projects.github.io/micronaut-validation/snapshot/guide/[Micronaut Validation] provides native support for the `javax.validation` annotations.
```

Comment:
extract this link out into properties as it will soon change from the snapshot

Reply:
after https://github.com/micronaut-projects/micronaut-validation/pull/49 is merged this will need to refer to `jakarta.validation`

Reply:
they are many instances of javax.validation, we will need to it in a separate PR.

Reply:
> extract this link out into properties as it will soon change from the snapshot

Done via: 
https://github.com/micronaut-projects/micronaut-core/pull/8876/commits/f0518a9219a0670853d7577eac05ff5ead62420a